circuitpython_library_pypi_stats.md

```diff
@@ -3,6 +3,10 @@
 Here is a listing of current Adafruit CircuitPython libraries download statistics.  
 There are 308 libraries available.
```

Comment:
These should be  a heading 2 (`##`). Please make the number libraries (`308` here) bold.